### PR TITLE
SG-1809 form confirmation page updates

### PR DESCRIPTION
**Pull request by aekong**
SG-1809 form confirmation page updates:
* disable banner color
* disable banner image
* add field_description (re-used field)
* add field_related_content (re-used-field)
* update form confirmation page header to use node title and new description field
* update header banner color to receive classname (intended for design system background color utility class)

### CODE REVIEW

**Review by arlina-espinoza — COMMENTED** [review]
Hi @aekong , this looks good! I just made a suggestion on a simpler way to set the page title.

File: web/themes/custom/sfgovpl/includes/page.inc [hunk 1]
```diff
@@ -28,6 +29,26 @@ function sfgovpl_preprocess_page__node__form_page(&$variables) {
...
+
+    // add the related form title to the variables array for use on template
+    if (!empty($node->get('field_related_content_single')->getValue())) {
+      $relatedForm = $node->get('field_related_content_single')->getValue();
+      $formNode = Node::load($relatedForm[0]['target_id']);
+      $variables['related_form_title'] = $formNode->getTitle(); 
+    }
+  }
+}
+
```
[the remark this answers is not in the document]
Comment (aekong) [reply web/themes/custom/sfgovpl/includes/page.inc:51]: ah yeah, the form confirmation page has its own title.  i'm trying to expose the title of a referenced node (form page) to put in the header of the form confirmation page

[the remark this answers is not in the document]
Comment (arlina-espinoza) [reply web/themes/custom/sfgovpl/includes/page.inc:51]: I meant the snippet to be:
```
/**
 * Preprocess page title.
 */
function THEMENAME_preprocess_page_title(&$variables) {
  if (
    ($node = \Drupal::routeMatch()->getParameter('node')) &&
    $node->bundle() == 'form_confirmation_page'
  ) {
    if ($relatedForm = $node->get('field_related_content_single')->entity) {
      $variables['title'] = $relatedForm->label();
    }
  }
}
```

[the remark this answers is not in the document]
Comment (aekong) [reply web/themes/custom/sfgovpl/includes/page.inc:51]: so i don't think our page layouts are using the default page title block, so there isn't a `page_title` hook available 😞 i'm trying to understand _why_ we're not using the default page title block and instead using `sfgovpl_page_title`.   @arlina-espinoza may reach out to pair with you on some drupal 101 things

**Review by arlina-espinoza — APPROVED** [review]
Note: we tried to make the page title preprocessor work without success, so I think @aekong 's approach here works nicely instead, just documenting that any other page alters can take this pattern.

**Comment by aekong** [comment]
thank you!

**Pull request merged**